package.json

```
@@ -11,20 +11,33 @@
     "range": "forta-agent run --range",
     "file": "forta-agent run --file",
     "publish": "forta-agent publish",
-    "test": "jest"
+    "test": "mocha ./src/bot-handlers/tests/liquidator-test.js && jest"
```

Let's tweak this so `test` will grab the disputer testfile as well:
```suggestion
    "test": "mocha ./src/bot-handlers/tests/*-test.js && jest"
```